interfaces/system_key.go

```diff
@@ -117,16 +117,16 @@ func generateSystemKey() (*systemKey, error) {
 	// profile.
 	sk.NFSHome, err = isHomeUsingNFS()
 	if err != nil {
+		// just log the error here
 		logger.Noticef("cannot determine nfs usage in generateSystemKey: %v", err)
-		return nil, err
 	}
 
 	// Add if '/' is on overlayfs so we can add AppArmor rules for
 	// upperdir such that if this changes, we change our profile.
 	sk.OverlayRoot, err = osutil.IsRootWritableOverlay()
 	if err != nil {
+		// just log the error here
```

Comment:
I think we should not do it here, if we do we can just skip returning errors again.

We should move this to `SystemKeyMismatch` and `WriteSystemKey` instead. especially as this has the same amount of lines to change.

Reply:
I was thinking about this, I'm not against it. The effect of that approach would be that people will an invalid fstab will always get re-generated  profiles. So slightly more expensive. OTOH more correct because we really don't know what to do (the system might be switching from nfs to !nfs or vice versa and we can't detect it).

Reply:
Pushed an update